Unity/Assets/YarnSpinner/Scripts/DialogueRunner.cs

```diff
@@ -127,8 +133,12 @@ public class StringUnityEvent : UnityEngine.Events.UnityEvent<string> { }
 
                     _dialogue.lineHandler = HandleLine;
                     _dialogue.commandHandler = HandleCommand;
-                    _dialogue.optionsHandler = HandleOptions;
-                    _dialogue.nodeCompleteHandler = (node) => { 
+                    _dialogue.optionsHandler = HandleOptions;
+					_dialogue.nodeStartHandler = (node) => {
+						onNodeStart?.Invoke(node, _dialogue.GetTagsForNode(node).ToArray());

```

Comment:
Can this be done without using LINQ?

Reply:
It'll not be as clean but it can be done. ^.^

Reply:
As a matter of fact, I think that this convertion from IEnumerable<string> to string[] is a waste, I'll just remove it and change the event associated to it.